plugin/main/src/kotlinx/benchmark/gradle/SuiteSourceGenerator.kt

```diff
@@ -208,15 +208,15 @@ class SuiteSourceGenerator(val title: String, val module: ModuleDescriptor, val
                     val annotation = it.annotations.findAnnotation(FqName(paramAnnotationFQN))!!
                     val constant = annotation.argumentValue("value") 
                         ?: error("@Param annotation should have at least one default value")
-                    val values = constant.value as? List<String>
+                    val values = constant.value as? List<StringValue>
                         ?: error("@Param annotation should have at least one default value")
                     values
                 })
                 
                 val defaultParametersString = defaultParameters.entries
-                    .joinToString(prefix = "mapOf(", postfix = ")") {  
-                        "\"${it.key}\" to " + it.value.joinToString(prefix = "listOf(", postfix = ")")
-                        
+                    .joinToString(prefix = "mapOf(", postfix = ")") { (key, value) ->
+                        "\"${key}\" to ${value.joinToString(prefix = "listOf(", postfix = ")") { "\"${it.value.replace("\"", "\\\"")}\"" }}"
```

Comment:
What about escaping other special characters?

Reply:
I have fixed it with the following commit: https://github.com/Kotlin/kotlinx-benchmark/commit/85ddb0655275e54a4ee62760dbed29db2f548650